openedx_webhooks/jira/tasks.py

```diff
@@ -5,11 +5,27 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+from collections import namedtuple
+
 from ..lib.jira.decorators import inject_jira
 from ..lib.jira.utils import (
     convert_to_jira_datetime_string, find_allowed_values, make_fields_lookup
 )
 
+GithubActivity = namedtuple('GithubActivity', [
+    'LAST_UPDATED_AT',
+    'LAST_UPDATED_BY',
+    'LATEST_ACTION',
+    'LATEST_BY_EDX',
+])
+
+GH_PR_ACTIVITY = GithubActivity(
+    LAST_UPDATED_AT='Github PR Last Updated At',
+    LAST_UPDATED_BY='Github PR Last Updated By',
+    LATEST_ACTION='Github Latest Action',
+    LATEST_BY_EDX='Github Latest Action by edX',
+)
```

Comment:
This seems like a long way to go when four constants would have sufficed, but ok.